### PR TITLE
finished fedora History

### LINKED ISSUE #5: [Fedora] Distribution History
- Release date.
  - Flavors. (if exist)
  - Based on distribution. (if exist) (example: Pop!_OS is based on Debian)

### PR DESCRIPTION
**Pull request by KenzyAdel**

### CODE REVIEW

File: Fedora/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#Fedora
```
Comment (thisisamna) [inline Fedora/README.md]: Need a space here between the # and the title

File: Fedora/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+![Fedora_LOGO](Imgs/Fedora_LOGO.png)
+---
+##Distribution History
```
Comment (thisisamna) [inline Fedora/README.md]: Space here too between # and title

File: Fedora/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+---
+Fedora Linux or ___"Fedora"___ is an **Open_source Linux** distribution that was released in **6 November 2003**. It was originally developed as a continuation of [Red Hat Linux](https://en.wikipedia.org/wiki/Red_Hat_Linux) project.
+Although fedora is not as stable as Red Hat Enterprise Linux (RHEL), but it is completely free to use and full of innovations, unlike Red Hat that is typically sold through an annual subscription.
```
Comment (thisisamna) [inline Fedora/README.md]: Awesome work. Capitalize the f of Fedora here

File: Fedora/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+##Distro Image
```
Comment (thisisamna) [inline Fedora/README.md]: Space here too

**Review by thisisamna — CHANGES_REQUESTED** [review]

File: Fedora/README.md (hunk 4 above)
Comment (thisisamna) [reply Fedora/README.md]: Note: The changes are minor and I'm not sure whether I should've merged and requested fixes later, but I'll play it safe

**Comment by KenzyAdel** [comment]
Spaces and redirecting all images

**Review by thisisamna — APPROVED** [review]
LGTM. Thank you Kenzy!

**Pull request merged**